packages/customize-widgets/src/components/inserter/inserter-outer-section.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
This file doesn't belong in `packages/customize-widgets/src/components` as it's not a React component. Maybe we could create a `views` folder since I think ultimately these things are backbone views.

Reply:
I put it here since it's more related to the inserter but I see what you mean.

Reply:
I don't think they're so tightly coupled that they need to live together.